web/app/templates/auth/email/reset_password.txt

```diff
@@ -1,13 +1,11 @@
-Dear {{ user.username }},
-
-To reset your password click on the following link:
+Kära {{ user.first_name }},
 
+Vänligen klistra in länken nedan i webbläsarens adressfält för att återställa ditt lösenord:
 {{ url_for('auth.password_reset', token=token, _external=True) }}
 
-If you have not requested a password reset simply ignore this message.
-
-Sincerely,
+Om du inte vill byta lösenord kan du ignorera detta meddelande.
 
-The Flasky Team
+Hälsningar,
+Chronos Development Team
 
-Note: replies to this email address are not monitored.
\ No newline at end of file
+Observera: svar till denna e-postadress ej blir lästa.
```

Comment:
This line should be "Observera: svar till denna e-postadress läses ej" or "Observera: svar till denna e-postadress blir ej lästa".